src/sentry/processing/issues_by_tag.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import absolute_import
+
+import six
+
+from sentry import tagstore
+from sentry.models import EventUser, Group, get_group_with_redirect
+from .base import BaseProcessor, ProcessingError
+
+
+class IssuesByTagProcessor(BaseProcessor):
+    def __init__(self, project_id, group_id, key, environment_id):
+        self.project = self.get_project(project_id)
+        self.group = self.get_group(group_id, self.project)
+        self.key = key
+        self.environment_id = environment_id
+        self.fields = self.get_fields(self.key)
+        self.lookup_key = self.get_lookup_key(self.key)
+        # Ensure the tag key exists, as it may have been deleted
+        try:
+            tagstore.get_tag_key(self.project.id, environment_id, self.lookup_key)
+        except tagstore.TagKeyNotFound:
+            raise ProcessingError("Requested key does not exist")
+        self.callbacks = self.get_callbacks(self.key, self.group.project_id)
```

Comment:
@leeandher do we need to bind the callbacks here are can we wait until we actually need to use them?

Reply:
I'm still binding the callbacks in the constructor (the file is `src/sentry/data_export/processors/issues_by_tag.py`), is there any reason not to? I just did it all in the constructor to keep each function cleaner